### PR TITLE
added jira support

### PR DESCRIPTION
**Pull request by osherdp**

### CODE REVIEW

**Comment by osherdp** [comment]
closes #147

File: src/store/config/index.js [hunk 1]
```diff
@@ -674,6 +678,21 @@ export default {
       return config;
     },
 
+    jira(state) {
+      let config = {};
+
+      if (state.alert.jiraProject && state.alert.jiraIssueType) {
+	config.jira_project = state.alert.jiraProject;
```
Comment (johnsusek) [inline src/store/config/index.js]: [Minor] Indentation

File: src/store/config/index.js [hunk 2]
```diff
@@ -674,6 +678,21 @@ export default {
...
+	config.jira_issuetype = state.alert.jiraIssueType;
+      }
+
+      if (state.alert.jiraComponents) {
+	config.jira_components = state.alert.jiraComponents;
```
Comment (johnsusek) [inline src/store/config/index.js]: [Minor] Indentation

**Comment by johnsusek** [comment]
Can you run `npm run test:unit` BTW? Looks like there might be commas missing from one file.

**Comment by osherdp** [comment]
I actually can't see why the build fails
I've done my best...

**Comment by johnsusek** [comment]
Yeah sorry the CI is a private server. I re-ran it and the checks pass now. I am deploying it then just going to make sure it looks good then I will merge and do a new release with this! Thanks for your contribution!

**Pull request merged**